test/unit/com/unifina/signalpath/blockchain/Web3jHelperSpec.groovy

```diff
@@ -26,28 +28,28 @@ class Web3jHelperSpec extends ModuleTestingSpecification {
 		byte[][][] twoDimBytesArray = [bytesArray, bytesArray, bytesArray]
 		Object[][] twoDimIntarray = [nums, nums, nums]
 
-		Type twoDarray = Web3jHelper.instantiateType("uint[][3]", twoDimIntarray)
+		Type twoDarray = TypeDecoder.instantiateType("uint[][3]", twoDimIntarray)
 
 		then:
-		Web3jHelper.instantiateType("bool", false) instanceof Bool
-		Web3jHelper.instantiateType("bool", 1) instanceof Bool
-		Web3jHelper.instantiateType("int", -123) instanceof Int
-		Web3jHelper.instantiateType("int256", -123) instanceof Int256
-		Web3jHelper.instantiateType("uint", 123) instanceof Uint
-		Web3jHelper.instantiateType("uint256", 123) instanceof Uint256
-		Web3jHelper.instantiateType("address", "0x123") instanceof Address
-		Web3jHelper.instantiateType("address", BigInteger.ONE) instanceof Address
-		Web3jHelper.instantiateType("uint256[]", Arrays.asList(nums)) instanceof DynamicArray
-		Web3jHelper.instantiateType("uint256[3]", Arrays.asList(nums)) instanceof StaticArray
-		Web3jHelper.instantiateType("uint256[]", nums) instanceof DynamicArray
-		Web3jHelper.instantiateType("uint256[3]", nums) instanceof StaticArray
-		Web3jHelper.instantiateType("bytes" + bytes.length, bytes) instanceof BytesType
-		Web3jHelper.instantiateType("bytes[]", Arrays.asList(bytesArray)) instanceof DynamicArray
-		Web3jHelper.instantiateType("bytes[3]", Arrays.asList(bytesArray)) instanceof StaticArray
-		Web3jHelper.instantiateType("uint[][]", twoDimIntarray) instanceof DynamicArray
-		Web3jHelper.instantiateType("uint[3][]", twoDimIntarray) instanceof DynamicArray
+		TypeDecoder.instantiateType("bool", false) instanceof Bool
```

Comment:
What's the class under test here, `Web3jHelper` or `TypeDecoder`? We shouldn't need to test `TypeDecoder`, as it's outside our codebase and presumably tested there. We only need to test the code in `Web3jHelper` (and `SolidityCompileDeploy`).